frontend/src/views/Layout/AuthLayout_gdd.vue

```diff
@@ -14,16 +14,16 @@
             </b-col>
             <b-col xl="6" class="col-xl-6">
               <b-nav class="nav-footer justify-content-center justify-content-lg-end">
-                <b-nav-item ref="https://gradido.net/" target="_blank">
+                <b-nav-item :href="'https://gradido.net/' + $i18n.locale" target="_blank">
                   Gradido
                 </b-nav-item>
-                <b-nav-item href="https://gradido.net/" target="_blank">
-                {{$t('imprint')}}
+                <b-nav-item :href="'https://gradido.net/' + $i18n.locale" target="_blank">
+                  {{$t('imprint')}}
                 </b-nav-item>
-                <b-nav-item href="https://gradido.net/" target="_blank">
+                <b-nav-item :href="'https://gradido.net/' + $i18n.locale" target="_blank">
                   {{$t('privacy_policy')}}
                 </b-nav-item>
-                <b-nav-item href="https://gradido.net/" target="_blank">
+                <b-nav-item :href="'https://gradido.net/' + $i18n.locale" target="_blank">
```

Comment:
I think here was a change merged. Please update this accordingly when resolving the conflict